tests/stream/test_ffmpegmux.py

```diff
@@ -51,22 +57,112 @@ def test_is_usable(self, session: Streamlink, resolved: Optional[str], expected:
         with patch("streamlink.stream.ffmpegmux.which", return_value=resolved):
             assert FFMPEGMuxer.is_usable(session) is expected
 
-    def test_log(self, session: Streamlink):
-        with patch("streamlink.stream.ffmpegmux.log") as mock_log, \
-             patch("streamlink.stream.ffmpegmux.which", return_value=None):
+    def test_log(self, caplog: pytest.LogCaptureFixture, session: Streamlink):
+        with patch("streamlink.stream.ffmpegmux.which", return_value=None):
             assert not FFMPEGMuxer.is_usable(session)
-            assert mock_log.warning.call_args_list == [
-                call("FFmpeg was not found. See the --ffmpeg-ffmpeg option."),
-                call("Muxing streams is unsupported! Only a subset of the available streams can be returned!"),
+            assert [(record.module, record.levelname, record.message) for record in caplog.records] == [
+                (
+                    "ffmpegmux",
+                    "warning",
+                    "No valid FFmpeg binary was not found. See the --ffmpeg-ffmpeg option.",
```

Comment:
Same typo here with the double negative.

Reply:
Copy pasted from the other one which I apparently didn't read after changing it :sweat_smile: